gtpv2/message/change-notification-req.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2019-2022 go-gtp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package message
+
+import "github.com/wmnsk/go-gtp/gtpv2/ie"
+
+// ChangeNotificationRequest is a ChangeNotificationRequest Header and its IEs above.
+type ChangeNotificationRequest struct {
+	*Header
+	IMSI                             *ie.IE
+	MEI                              *ie.IE
+	IndicationFlags                  *ie.IE
+	RATType                          *ie.IE
+	ULI                              *ie.IE
+	UCI                              *ie.IE
+	PGWS5S8IPAddressForControlPlane  *ie.IE
+	LinkedEBI                        *ie.IE
+	PresenceReportingAreaInformation []*ie.IE
+	MOExceptionDataCounter           *ie.IE
+	SecondaryRATUsageDataReport      []*ie.IE
+	PrivateExtension                 *ie.IE
```

Comment:
https://www.etsi.org/deliver/etsi_ts/129200_129299/129274/17.06.00_60/ts_129274v170600p.pdf

I have found also `PSCell ID` field. Please re-check if it is needed here

Reply:
Too new, needs many changes, out of scope